encoders/numeric/CompressionVaeEncoder/__init__.py

```diff
@@ -68,16 +76,10 @@ def encode(self, data: 'np.ndarray', *args, **kwargs) -> 'np.ndarray':
         :param data: a `B x T` numpy ``ndarray``, `B` is the size of the batch
         :return: a `B x D` numpy ``ndarray``
         """
-        from cvae import cvae
-        model = cvae.CompressionVAE(X=self.X,
-                                    train_valid_split=self.train_valid_split,
-                                    dim_latent=self.output_dim,
-                                    iaf_flow_length=self.iaf_flow_length,
-                                    cells_encoder=self.cells_encoder,
-                                    initializer=self.initializer,
-                                    batch_size=self.batch_size,
-                                    batch_size_test=self.batch_size_test,
-                                    logdir=self.model_path,
-                                    feature_normalization=self.feature_normalization,
-                                    tb_logging=self.tb_logging)
-        return model.embed(data)
+        import tensorflow as tf
+
+        return self.sess.run([self.embeddings],
+                             feed_dict={self.data_feature_placeholder: data})[0]
+
+    def close(self) -> None:
+        self.sess.close()
```

Comment:
Can u add a super.close() call to ensure anything added on base classes' close is also executed?

Reply:
Sure